hackathon_site/event/jinja2/event/landing.html

```diff
@@ -116,7 +116,7 @@ <h4 class="center white-text">Saturday</h4>
                         </p>
                     </div>
                     <div class="scheduleRow">
-                        <p class="lightText scheduleTime">Various</p>
+                        <p class="lightText scheduleTime">2PM</p>
```

Comment:
@EthanHugh Could you match all the times on the Figma exactly? I'm not sure what logistics wants this year but assuming they think the Figma is what's going to be on the website we should follow it exactly.
Right now the schedule is copied from last year so it's missing some times (e.g. Participant Registration @ 9am). Everything else looks great!